src/libraries/System.Private.CoreLib/src/System/Globalization/Ordinal.cs

```diff
@@ -277,6 +241,29 @@ internal static int IndexOfOrdinalIgnoreCase(ReadOnlySpan<char> source, ReadOnly
             return OrdinalCasing.IndexOf(source, value);
         }
 
+        internal static int LastIndexOfNoIgnoreCase(string source, string value, int startIndex, int count)
```

Comment:
_NoIgnoreCase_ reads awkwardly. Consider dropping it entirely (since default is case-sensitive) or using _CaseSensitive_ as a suffix.